pysat/_instrument.py

```diff
@@ -1565,8 +1564,8 @@ def download(self, start=None, stop=None, freq='D', user=None,
             # longer than a day then the download defaults would
             # no longer be correct. Dates are always correct in this
             # setup.
-            logger.info('Downloading the most recent data by default ' +
-                  '(yesterday through tomorrow).')
+            logger.info('Downloading the most recent data by default '
+                        + '(yesterday through tomorrow).')
```

Comment:
```suggestion
            logger.info(''.join(['Downloading the most recent data by ',
                                           'default (yesterday through tomorrow).']))
```

Reply:
Will need to check whitespace, since I am not sure the suggestion box is good for getting the correct alignment.